index.html

```diff
@@ -6,6 +6,423 @@
+
+	<!-- Start solution block -->
+	<section class="solution">
+		<header class="solution__header">
+			<img src="assets/florence.svg" alt="">
+			<h2>
+				<span class="small-title">The Concept</span>
+				Meet Florence
+			</h2>
+			<p>
+				Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen.  A chatbot at heart, patients would converse with her through text and picture messages. 
+			</p>
+			<p>
+				If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.
+			</p>
+		</header>
+
+		<!-- nest sections within the meetFlorence section -->
+		<!-- these sections will be styled as white boxes and share a common class -->
+
+		<section class="solution__explanation">
+		<!-- the firstHalf and secondHalf classes will both be on the same line, side-by-side -->
+
+			<header>
+				<h3>She Gets To Know You.</h3>
+				<p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+				<p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
+			</header>
+
+			<section class="solution__chatbox">
+
+				<ul class="chatbox">
+					<li class="chatbox__chat--left">
+						Hi! I'm Florence. What's your name?
+					</li>
+					<li class="chatbox__chat--right">
+						Mike
+					</li>
+					<li class="chatbox__chat--left">
+						Nice to meet you, Mike. I can help you set up your account with a few simple questions.
+					</li>
+					<li class="chatbox__chat--left">
+						You can answer these now or whenever you like. I'm always here!
+					</li>
+					<li class="chatbox__chat--left">
+						<img src="assets/florence-avatar.svg" alt="">
+						First question: how old are you?
+					</li>
+					<li class="chatbox__chat--right">
+						65
+					</li>
+				</ul>
+
+			</section>
+
+			<!-- this image must have an absolute position -->
+			<img src="assets/section1-illo.svg" alt="" id="section1-illo">
+		</section>
+
+		<!-- note the above format works well for the rest of the chat bubbles -->
+
+		<section class="solution__explanation">
+
+			<header>
+				<h3>She Keeps Track of Your Regimen.</h3>
+				<p>Florence helps you remember what to take and when. Image recognition makes adding your medication as easy as taking a picture.</p>
+			</header>
+
+			<section class="solution__chatbox">
+
+				<ul class="chatbox">
+					<li class="chatbox__chat--left">
+						Let's get started adding your medications.
+					</li>
+					<li class="chatbox__chat--left">
+						Message me a picture of the bottle label and I'll add it. You can also type in the name if you want.
+					</li>
+					<li class="chatbox__chat--right">
+						<img src="assets/sample-photo.png" alt="">
```

Comment:
These images are content, so images in the chat should have more descriptive `alt` text